exporter/clients.go

```diff
@@ -47,17 +51,17 @@ func parseClientListString(clientInfo string) ([]string, bool) {
 		return nil, false
 	}
 
-	return []string{
-		connectedClient["name"],
-		createdAtTs,
-		idleSinceTs,
-		connectedClient["flags"],
-		connectedClient["db"],
-		connectedClient["omem"],
-		connectedClient["cmd"],
-
-		hostPortString[0], // host
-		hostPortString[1], // port
+	return &ClientInfo{
+		Name:       connectedClient["name"],
```

Comment:
You can get rid of `connectedClient` and directly populate the ClientInfo struct